### ISSUE TITLE
The configuration parameter "componentType" is a required for "" component.

### ISSUE THREAD

**Issue by patrickdeamorim** [Yoast/Yoast-SEO-for-Magento2#63]
I get this error upon installing this module on Magento 2.2. I installed via Composer (and also tried manually after it did not work).

This happens when going into a product in the admin. I went into a product minutes before installing the extension without issues, and the issue only appeared after the extension was installed.

The full error message is:
https://gist.github.com/patrickdeamorim/7a979108f8c631fe6b8469ae54ce3f8f

**Comment by govereem** [comment]
Error comes from MaxServ\YoastSeo\Ui\Catalog\DataProvider\Product\Form\Modifier\YoastSeo.php
```
        $meta[$groupCode]['children']['container_url_key']['children']['url_key']['arguments']['data']['config']['additionalClasses'] = 'yoastBox-urlKey';
        $meta[$groupCode]['children']['container_meta_title']['children']['meta_title']['arguments']['data']['config']['additionalClasses'] = 'yoastBox-metaTitle hidden';
        $meta[$groupCode]['children']['container_meta_description']['children']['meta_description']['arguments']['data']['config']['additionalClasses'] = 'yoastBox-metaDescription hidden';
        $meta[$groupCode]['children']['container_meta_keyword']['children']['meta_keyword']['arguments']['data']['config']['additionalClasses'] = 'yoastBox-metaKeywords';
        $meta[$groupCode]['children']['container_focus_keyword']['children']['focus_keyword']['arguments']['data']['config']['additionalClasses'] = 'yoastBox-focusKeyword';
```

**Comment by cfgalvani** [comment]
Any fix for it?

**Comment by vincenthornikx** [comment]
@patrickdeamorim @govereem @cfgalvani this issue should be fixed now in the 2.0.0 release
